content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+
+  Issue the following command to download the Palette Edge CLI. This command uses `curl` to download the Palette Edge CLI binary from the specified URL and save it as **palette-edge** in the current directory. 
+<br />
+
+  ```bash
+  curl https://software.spectrocloud.com/stylus/v3.4.3/cli/linux/palette-edge --output palette-edge
+  ```
+
+
+2. Set the executable permissions for the **palette-edge** binary by issuing the following command.
+<br />
+
+  ```bash
+  chmod 755 palette-edge
+  ```
+
+
+3. Use the following command to move the **palette-edge** binary to the **/bin/** directory. You will need elevated privileges to perform this operation. This step will allow you to issue the `palette-edge` command from any directory in your development environment.
+<br /> 
+
+  ```bash
+  sudo mv palette-edge /bin/
+  ```
+
+
+4. Verify the installation of the Palette Edge CLI by issuing the following command. The output will display information about the installed version of the Palette Edge CLI. 
+<br />
+
+  ```bash
+  palette-edge show
+  ```
+
+  ```bash hideClipboard
+  # Sample output   
+  ┌────────────────────────────────────────────────────────────────────────┐
+  | OS Flavor     | Description        | Base Image URI                    |
+  | opensuse-leap | Opensuse Leap 15.4 | quay.io/kairos/core-opensuse-leap |
+  | ubuntu-20     | Ubuntu 20.4 LTS    | quay.io/kairos/core-ubuntu-20-lts |
+  | ubuntu-22     | Ubuntu 22.4 LTS    | quay.io/kairos/core-ubuntu-22-lts |
+  └────────────────────────────────────────────────────────────────────────┘
+  ┌─────────────────────────────────────────────────────────────────────────────────────────────┐
+  | K8S Flavor | Description        | Supported Versions                                        |
+  | k3s        | Rancher K3s        | 1.25.2-k3s1,1.24.6-k3s1,1.23.12-k3s1,1.22.15-k3s1         |
+  | kubeadm    | Kubernetes kubeadm | 1.25.2,1.24.6,1.23.12,1.22.15                             |
+  | rke2       | Rancher RK2        | 1.25.2-rke2r1,1.24.6-rke2r1,1.23.12-rke2r1,1.22.15-rke2r1 |
+  └─────────────────────────────────────────────────────────────────────────────────────────────┘
+  ┌─────────────────────────────────┐
+  | Component             | Version |
+  | Spectro Agent Version | v3.4.3  |
+  | Kairos Version        | v2.0.3  |
+  └─────────────────────────────────┘
+  ```
+<br />
+
+
+5. The next few steps will guide you to create a cluster profile referencing the provider image you must have already pushed to an image registry. Check the **Prerequisites** section if you do not have a provider image in an image registry. 
+
+  Open a web browser and log in to [Palette](https://console.spectrocloud.com). Ensure you are in the **Default** project scope before creating a cluster profile. 
```

Comment:
```suggestion
5. Open a web browser and log in to [Palette](https://console.spectrocloud.com). Ensure you are in the **Default** project scope before creating a cluster profile. 
```

This is not a tutorial. Go to the next steps. There is no need for this verbosity when you provided an overview in the beginning. How-tos are concise and to the point.